libs/checkpoint/langgraph/checkpoint/sqlite/__init__.py

```diff
@@ -132,20 +133,22 @@ def setup(self) -> None:
             PRAGMA journal_mode=WAL;
             CREATE TABLE IF NOT EXISTS checkpoints (
                 thread_id TEXT NOT NULL,
-                checkpoint_ns TEXT NOT NULL,
+                checkpoint_ns TEXT NOT NULL DEFAULT '',
                 checkpoint_id TEXT NOT NULL,
                 parent_checkpoint_id TEXT,
+                type TEXT,
```

Comment:
I think in this version without blobs table you'd need a checkpoint_type and metadata_type?

Reply:
Or maybe better to serialize metadata always with `json.dumps`

Reply:
updated w/ `dumps`!